backend/src/test/java/botobo/core/documentation/UserDocumentationTest.java

```diff
@@ -120,4 +125,30 @@ void checkSameUserNameAlreadyExist() throws Exception {
                 .status(status().isOk())
                 .identifier("users-name-check-post-success");
     }
+
+    @DisplayName("문제집명을 포함한 문제집의 유저를 모두 가져온다. - 성공")
+    @Test
+    void findAllUsersByWorkbookName() throws Exception {
+        // given
+        final User 조앤 = User.builder()
+                .id(1L)
+                .userName("조앤")
+                .build();
+        final User 민정 = User.builder()
+                .id(1L)
```

Comment:
```suggestion
                .id(2L)
```
사소한 부분이지만, 문서에서 보면 다른 사람인데 같은 id를 가지고 있기 때문에
보는 사람이 혼동이 올 수 있다고 생각해요.

Reply:
복붙했더니 이렇게 된지도 몰랐네요 ㅠ_ㅠ_ㅠ 바로 수정했슴당